packages/fast-components-class-name-contracts/src/index.ts

```diff
@@ -1,6 +1,7 @@
 export * from "./button";
 export * from "./divider";
 export * from "./image";

```

Comment:
I mean - we can correct this later if need bew, but technically "h" would be before "i" :)